docs/tutorials/first-run-experience-tutorial.md

```diff
@@ -0,0 +1,150 @@
+                </li>
+                <li class="ms-ListItem">
+                    <i class="ms-Icon ms-Icon--Design ms-font-xl"></i>
+                    <span class="ms-font-m">Create and visualize like a pro</span>
+                </li>
+            </ul>
+        </div>
+        <p class="ms-font-l">Select any range of cells in the worksheet, then click <b>Run</b>.</p>
+        <div role="button" id="run" class="ms-welcome__action ms-Button ms-Button--hero ms-font-xl">
+            <span class="ms-Button-label">Run</span>
+        </div>
+        <p><label id="item-subject"></label></p>    
+    </main>
+    ```
+
+1. If you selected an Office application besides Excel, update the first `<p>` element with more appropriate instructions.
+
+### Update the JavaScript file
+
+Update the JavaScript file to display the first-run experience if this is the first time the user is running the add-in.
+
+1. Open the **taskpane.js** file. Replace the `Office.onReady` statement with the following code, then save the file. Some notes about this code:
+
+    - It checks local storage for a key called "showedFRE". If the key doesn't exist, then show the first-run experience.
+    - It adds a new function called `showFirstRunExperience` that displays the "first-run-experience" `<div>` added to the HTML. This function also adds the "showedFRE" item to local storage.
+
+    ```javascript
+    Office.onReady((info) => {
+      if (info.host === Office.HostType.Excel) {
+        document.getElementById("sideload-msg").style.display = "none";
+        document.getElementById("app-body").style.display = "flex";
+
+        // showedFRE is created and set to "true" when you call showFirstRunExperience().
+        if (!localStorage.getItem("showedFRE")) {
+          showFirstRunExperience();
+        }
+    
+        document.getElementById("run").onclick = run;
+      }
+    });
+    
+    async function showFirstRunExperience() {
+      document.getElementById("first-run-experience").style.display = "flex";
+      localStorage.setItem("showedFRE", true);
+    }  
+    ```
+
+1. If you selected an Office application besides Excel, update the condition of the first `if` statement to check for your chosen [Office.HostType](/javascript/api/office/office.hosttype).
+
+### Update the CSS file
+
+Update the CSS file to ensure that the add-in UI is styled appropriately given the addition of the "first-run-experience" `<div>`.
+
+- Open the **taskpane.css** file. Replace the line `.ms-welcome__main {` with the following code, then save the file.
+
+    ```css
+    .ms-welcome__main, .ms-welcome__main > div {
+    ```
+
+### Try it out
+
+1. Ensure that the web server is running and the add-in has been sideloaded. For details, see the instructions in the quick start you used.
+
+1. If the add-in task pane isn't already open in Office, go to the **Home** tab and choose the **Show Taskpane** button on the ribbon to open it.
```

Comment:
Consider adding a note here for Outlook since the location of the add-in varies depending on the client. Maybe we can refer the user back to the quick start or to [Use add-ins in Outlook](https://support.microsoft.com/office/1ee261f9-49bf-4ba6-b3e2-2ba7bcab64c8) for instructions.